01_getting_started/hello_world.py

```diff
@@ -86,16 +83,16 @@ def main():
 # ## What just happened?
 #
 # When we called `.remote` on `f`, the function was executed
-# **in the cloud**, on Modal's infrastructure, not locally on our computer.
+# _in the cloud_, on Modal's infrastructure, not locally on our computer.
 #
 # In short, we took the function `f`, put it inside a container,
 # sent it the inputs, and streamed back the logs and outputs.
 #
 # ## But why does this matter?
 #
-# Try doing one of these things next to start seeing the full power of Modal!
+# Try doing one of these things next to see the full power of Modal.
```

Comment:
Why imply that the full power is subsumed by doing just one of the things below?

And also, I think that talking about the "full power" of our platform is an apposite time for an exclamation point.

Reply:
Just trying to flow better into the next paragraph: less words, and don't stop here